bears/r/RLintBear.py

```diff
@@ -23,7 +23,8 @@ class RLintBear:
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     REQUIREMENTS = {RscriptRequirement(package='lintr', flag='-e',
-                                       repo='http://cran.rstudio.com')}
+                                       repo='http://cran.rstudio.com',
+                                       version='>=3.1.1')}
```

Comment:
![capture](https://cloud.githubusercontent.com/assets/16348426/23104129/f4e4fdba-f6ed-11e6-9d88-ef2cb4d14f34.PNG)

Is this the one?